src/main/resources/templates/mdBenefitsFlow/redirectToMDThink.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html th:lang="${#locale.language}">
+<head th:replace="~{fragments/head :: head(title=#{redirect.mdthink.title})}"></head>
+<body>
+<div class="page-wrapper">
+  <div th:replace="~{fragments/toolbar :: toolbar}"></div>
+  <section class="slab">
+    <div class="grid">
+      <main id="content" role="main" class="form-card spacing-above-35">
+        <th:block th:replace="~{fragments/icons :: nextSteps}"/>
+        <th:block
+            th:replace="~{fragments/cardHeader :: cardHeader(header=#{redirect.mdthink.header})}"/>
+        <th:block>
+            <div class="form-card__content">
+               <a th:text="#{redirect.mdthink.linktext}"
+                  href="https://identity.mymdthink.maryland.gov/identityiq/external/portal-registration.jsf?gotoUrl=https://mymdthink.maryland.gov/cp/"
```

Comment:
The index page uses "https://mymdthink.maryland.gov/home/#/". See https://github.com/codeforamerica/mdpilot/blob/main/src/main/resources/messages.properties#L44.
We might want to find out what they prefer.

Reply:
Hmmm. I thought I used what was in figma, but I think I either did it wrong, or it changed. It's `https://mymdthink.maryland.gov/home/#/` so I'll change this link to that.

Reply:
I'll reuse the message that's displayed on the first page, since the wording is the same.